openapi/swagger/fs/usr/share/nginx/html/helper/index.html

```diff
@@ -139,6 +142,9 @@ <h4>Import/curate data</h4>
           <li>
             <a href="/swagger/?urls.primaryName=manageData">manageData</a> - API endpoints for managing data
           </li>
+          <li>
+              <a href="/swagger/?urls.primaryName=filesCurator">filesCurator</a> - API endpoints for managing files
+          </li>
```

Comment:
```suggestion
              <a href="/swagger/?urls.primaryName=filesCurator">filesCurator</a> - API endpoints for managing files, attached to a study
          </li>
```
This is a my suggestion, but let's approve with product guys